.github/workflows/post_release_version_bump.yml

```
@@ -1,8 +1,9 @@
 name: Automated Release Version Bumps
 
 on:
-  release:
-    types: [published]
+    push:
+        branches:
+            - patch_vb_script
```

Done testing the patch
```suggestion
    release:
        types: [published]
```